packages/shared/src/components/modals/award/BuyCoresModal.tsx

```diff
@@ -354,6 +354,18 @@ const BuyCoresMobile = ({ amountNeededCopy }: BuyCoresProps) => {
     }
   }, [openCheckout, selectedProduct, paddle]);
 
+  if (iOSSupportsCoresPurchase()) {
+    return (
+      <ModalBody
+        className={classNames(
+          'bg-gradient-to-t from-theme-overlay-float-bun to-transparent',
+        )}
+      >
+        <CoreOptions amountNeededCopy={amountNeededCopy} />
```

Comment:
On iOS checkout is a native popup and because of that on iOS we always keep rendering options

Reply:
<img width="458" height="923" alt="image" src="https://github.com/user-attachments/assets/6a0f6e69-6bb5-4bd7-8dcf-5f1c2297ad15" />